docs/UserGuide.md

```diff
@@ -86,17 +86,17 @@ This user guide will help you get started and understand how FABook can **seamle
   e.g. if the command specifies `help 123`, it will be interpreted as `help`.
  </div>
 
-### General
+## General
```

Comment:
I think we shld leave this as ### instead of ## since the feature header itself is ## already

Reply:
^